### PR TITLE
feat(web-app): add typescaletable & typeweight mdx components

### LINKED ISSUE #615: TypeScaleTable & TypeWeight MDX components
https://carbondesignsystem.com/guidelines/typography/overview#scale
`<TypeScaleTable />`

https://carbondesignsystem.com/guidelines/typography/overview#weights
```
<TypeWeight type="types" />
<TypeWeight />
<TypeWeight type="italic" />
```

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #615 

add TypeScaleTable & TypeWeight components
add text token to accordion & anchorlinks so they work with theme switching in storybook

#### Testing / reviewing

http://localhost:3000/mdx/type
http://localhost:6006/?path=/docs/platform-components-typeweight--default
http://localhost:6006/?path=/docs/platform-components-typescaletable--default

https://carbondesignsystem.com/guidelines/typography/overview#scale
https://carbondesignsystem.com/guidelines/typography/overview#weights

### CODE REVIEW

File: services/web-app/components/type/type-scale.module.scss [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    @include breakpoint.breakpoint('xlg') {
+      padding-right: spacing.$spacing-09;
+    }
+  }
+
+  @for $i from 1 through length($type-scale) {
+    table tbody tr:nth-child(#{$i}) td:nth-child(2) {
+      width: 100%;
+      /* stylelint-disable scss/no-global-function-names */
+      font-size: nth($type-scale, $i);
```
Comment (jdharvey-ibm) [inline services/web-app/components/type/type-scale.module.scss:74]:
```scss
@use 'sass:list' as list;

font-size: list.nth($type-scale, $i);
```

**Review by jharvey10 — CHANGES_REQUESTED** [review]

File: services/web-app/components/type/type-scale.module.scss (hunk 1 above)
Comment (alisonjoseph) [reply services/web-app/components/type/type-scale.module.scss:74]: This isn't working at all now with the latest update 😿

Comment (alisonjoseph) [reply services/web-app/components/type/type-scale.module.scss:74]: I'm going to simplify this file, having it set up this way doesn't make it any easier to understand really.

Comment (alisonjoseph) [reply services/web-app/components/type/type-scale.module.scss:74]: nvm, ignore me 😆

**Review by jharvey10 — APPROVED** [review]

File: services/web-app/stories/components/type-weight.stories.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+export const Italic = Template.bind({})
+Italic.args = {
+  type: 'italic'
+}
+
+export const Types = Template.bind({})
+Types.args = {
+  type: 'types'
+}
```
Comment (francinelucca) [inline services/web-app/stories/components/type-weight.stories.js:31]: Do we need to have these three stories if there's a control to switch the type?
image

**Review by francinelucca — APPROVED** [review]

**Pull request merged**